docs/adding games.md

```diff
@@ -6,6 +6,11 @@ There are two key steps to incorporating a game into Archipelago:
 - Game Modification 
 - Archipelago Server Integration
 
+Refer to the following documents as well:
+    * [network protocol.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/network%20protocol.md)
+    * [api.md](https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/api.md)
```

Comment:
+1, but maybe give them a title? like `[network protocol.md] for network communication detals`